indigo_app/static/javascript/indigo/views/work_commencements.js

```diff
@@ -60,6 +64,24 @@
       $(e.target).closest('.card').find('.provisions-commenced').toggleClass('d-none', e.target.checked);
     },
 
+    onExpandCollapseClick: function(e) {
+      e.stopPropagation();
+      const button = e.currentTarget;
+      const parent = button.closest('li');
+      if(button.classList.contains('expanded')) {
```

Comment:
`button.classList.toggle('expanded')`

Please also put a space in `if(` (and below, as well).

Reply:
done